app/assets/scripts/components/project-timeline.js

```diff
@@ -26,14 +26,12 @@ var ProjectTimeline = React.createClass({
 
     return (
       <div className='timeline'>
-        {actualStart && actualEnd && (
-          <div className='timeline__unit'>
-            <h5>Current Progress</h5>
-            {timeline(actualStart, actualEnd, scale)}
-          </div>
-        )}
         <div className='timeline__unit'>
-          <h5>Proposed Timeline</h5>
+          <h5 className='timeline__headline heading-alt'>Current Progress</h5>
```

Comment:
@ascalamogna this changes it so that the "Current Progress" header will show even if the current progress timeline is hidden (because the project hasn't started yet or there's no data for it, for example). Just making sure that's what we want here.